File: app/src/main/java/com/project/scratchstudio/kith_andoid/network/model/user/UserListResponse.java
package com.project.scratchstudio.kith_andoid.network.model.user;

import com.google.gson.annotations.SerializedName;
import com.project.scratchstudio.kith_andoid.network.model.BaseResponse;

import java.util.ArrayList;

public class UserListResponse extends BaseResponse {

    @SerializedName("users")
    private ArrayList<User> users;

    public ArrayList<User> getUsers() {
        return users;
    }

}

File: app/src/main/java/com/project/scratchstudio/kith_andoid/network/model/user/UserListResponse.kt
package com.project.scratchstudio.kith_andoid.network.model.user

import com.google.gson.annotations.SerializedName
import com.project.scratchstudio.kith_andoid.network.model.BaseResponse

import java.util.ArrayList

class UserListResponse : BaseResponse() {

    @SerializedName("list")
    lateinit var users: ArrayList<User>

}
